src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/AttributesAccordion.tsx

```diff
@@ -55,11 +60,36 @@ const AttributesAccordion = (props: Props) => {
     props.setAttributesAccordionExpandedPanel(newExpandedPanels[0]);
   };
 
+  const onReset = () => {
+    props.resetSelectedAttributes();
+    customDownloadStorageService.saveSelectedAttributes({});
+  };
+
+  const buildSection = (
+    section: string,
+    hideTitles?: boolean,
+    hideUnchecked?: boolean
```

Comment:
can we, instead of two boolean flags passed (sometimes) as individual parameters `buildSection('genes')`, `buildSection('genes', true, true)` use one argument for `buildSection` which is an object of shape:

```
{
  section: string, // or perhaps enum
  hideTitles?: boolean,
  hideUnchecked?: boolean
}
```

Reply:
That's done